src/parser.cpp

```diff
@@ -0,0 +1,89 @@
+#include <iostream>
+#include <iomanip>
+#include <sstream>
+#include <fstream>
+#include <nlohmann/json.hpp>
+#include <dcmtk/dcmpstat/dcmpstat.h>
+
+using json = nlohmann::json;
+
+using namespace std;
+
+
+class Instances {
+
+    public:
+        void initial_settings();
```

Comment:
can you add comments giving a description of what you think each of these functions should do? it doesn't matter if you haven't implemented all the functionality yet, just give the plan. Ideally in doxygen style (https://www.doxygen.nl/manual/docblocks.html), but this isn't essential at this stage